tests/utils.ts

```diff
@@ -84,4 +84,9 @@ export const getBranch = () =>
     })
   })
 
+export const waitForSkeletonsToHide = async (page: Page) => {
```

Comment:
If a new skeleton appears while it waits for the existing skeletons to hide, I think that it won't wait for the new skeleton